tests/functional/gaze_file_processing_test.py

```diff
@@ -53,117 +56,83 @@ def fixture_gaze_init_kwargs(request):
-            'experiment': pm.Experiment(1024, 768, 38, 30, 60, 'center', 1000),
+            'experiment': Experiment(1024, 768, 38, 30, 60, 'center', 1000),
         },
         'csv_binocular': {
             'file': 'tests/files/binocular_example.csv',
             'time_column': 'time',
             'time_unit': 'ms',
             'pixel_columns': ['x_left_pix', 'y_left_pix', 'x_right_pix', 'y_right_pix'],
             'position_columns': ['x_left_pos', 'y_left_pos', 'x_right_pos', 'y_right_pos'],
-            'experiment': pm.Experiment(1024, 768, 38, 30, 60, 'center', 1000),
+            'experiment': Experiment(1024, 768, 38, 30, 60, 'center', 1000),
         },
         'ipc_monocular': {
             'file': 'tests/files/monocular_example.feather',
-            'experiment': pm.Experiment(1024, 768, 38, 30, 60, 'center', 1000),
+            'experiment': Experiment(1024, 768, 38, 30, 60, 'center', 1000),
         },
         'ipc_binocular': {
             'file': 'tests/files/binocular_example.feather',
-            'experiment': pm.Experiment(1024, 768, 38, 30, 60, 'center', 1000),
+            'experiment': Experiment(1024, 768, 38, 30, 60, 'center', 1000),
         },
         'eyelink_monocular': {
             'file': 'tests/files/eyelink_monocular_example.asc',
-            'experiment': pm.DatasetLibrary.get('ToyDatasetEyeLink').experiment,
+            'definition': datasets.ToyDatasetEyeLink(),
         },
         'eyelink_monocular_2khz': {
             'file': 'tests/files/eyelink_monocular_2khz_example.asc',
-            'experiment': pm.Experiment(
+            'experiment': Experiment(
                 1280, 1024, 38, 30.2, 68, 'upper left',
-                eyetracker=pm.EyeTracker(
+                eyetracker=EyeTracker(
                     sampling_rate=2000.0, left=True, right=False,
                     model='EyeLink Portable Duo', vendor='EyeLink',
                 ),
             ),
         },
         'eyelink_monocular_no_dummy': {
             'file': 'tests/files/eyelink_monocular_no_dummy_example.asc',
-            'experiment': pm.Experiment(
+            'experiment': Experiment(
                 1920, 1080, 38, 30.2, 68, 'upper left',
-                eyetracker=pm.EyeTracker(
+                eyetracker=EyeTracker(
                     sampling_rate=500.0, left=True, right=False,
                     model='EyeLink 1000 Plus', vendor='EyeLink',
                 ),
             ),
         },
         'didec': {
             'file': 'tests/files/didec_example.txt',
-            'time_column': pm.DatasetLibrary.get('DIDEC').time_column,
-            'time_unit': pm.DatasetLibrary.get('DIDEC').time_unit,
-            'pixel_columns': pm.DatasetLibrary.get('DIDEC').pixel_columns,
-            'experiment': pm.DatasetLibrary.get('DIDEC').experiment,
-            **pm.DatasetLibrary.get('DIDEC').custom_read_kwargs['gaze'],
+            'definition': datasets.DIDEC(),
```

Comment:
adding these lines makes it harder and harder to get rid of datasets.PUBLICDATASET

Reply:
don't worry, we will get rid of the class implementations without any breaking change by forwarding to the `DatasetLibrary` item for each dataset: `pymovements.datasets.DIDEC = pymovements.DatasetLibrary.get('DIDEC')`.

Reply:
I don't expect any user to use `DatasetLibrary` ever at all.